pkg/apis/v2beta2/receiver_types.go

```diff
@@ -201,6 +201,32 @@ type SmsReceiver struct {
 	Template *string `json:"template,omitempty"`
 }
 
+// PushoverUserProfile includes userKey and other preferences
+type PushoverUserProfile struct {
+	// UserKey is the user (Pushover User Key) to send notifications to.
+	UserKey *string `json:"userKey"`
+	// Devices refers to device name to send the message directly to that device, rather than all of the user's devices
+	Devices []string `json:"devices,omitempty"`
+	// Title refers to message's title, otherwise your app's name is used.
+	Title *string `json:"title,omitempty"`
+	// Sound refers to the name of one of the sounds (https://pushover.net/api#sounds) supported by device clients
+	Sound  *string `json:"sound,omitempty"`
+}
+
+type PushoverReceiver struct {
+	// whether the receiver is enabled
+	Enabled *bool `json:"enabled,omitempty"`
+	// PushoverConfig to be selected for this receiver
+	PushoverConfigSelector *metav1.LabelSelector `json:"pushoverConfigSelector,omitempty"`
+	// Selector to filter alerts.
+	AlertSelector *metav1.LabelSelector `json:"alertSelector,omitempty"`
+	// The name of the template to generate DingTalk message.
+	// If the global template is not set, it will use default.
+	Template             *string `json:"template,omitempty"`
+	// The users profile.
+	Profiles []*PushoverUserProfile `json:"profiles,omitempty"`
```

Comment:
If the `Profiles` must be not nil, the `omitempty` should be deleted.

Reply:
Resolved: https://github.com/txfs19260817/notification-manager/blob/b6387586c55f6e2581fe431be82374a9b994be37/pkg/apis/v2beta2/receiver_types.go#L228